src/main/java/gov/cms/mat/cql_elm_translation/service/HumanReadableService.java

```diff
@@ -269,7 +269,7 @@ List<HumanReadablePopulationModel> buildMeasureObservation(
   List<HumanReadableExpressionModel> buildDefinitions(Set<CQLDefinition> allDefinitions) {
     List<CQLDefinition> definitions =
         allDefinitions.stream()
-            .filter(definition -> definition.getParentLibrary() == null)
+            .filter(definition -> definition.getParentLibrary() == null && !definition.isFunction())
```

Comment:
I believe this is a good place to have a unit test case so that we don't incidentally revert this filter.

Reply:
Added